lib/screens/visits_screen/visits_screen.dart

```diff
@@ -213,7 +213,11 @@ class _VisitsScreenState extends State<VisitsScreen> with PageViewNavigation {
 
   void _previousPageCallback() {
     setState(() {
-      _nextDisabled = _disabledNextPages[page.toInt() - 1];
+      if (page.toInt() - 1 < 0) {
+        _nextDisabled = false;
```

Comment:
Why do you set _nextDisabled to false?

Reply:
i need to think about this, what i did here, then I will answer again, because i think this is wrong.

Reply:
Sorry, my mistake.
I changed it to:
```
      if (page.toInt() >= 1) {
        _nextDisabled = _disabledNextPages[page.toInt() - 1];
      }
``` 
so:
     
![image](https://github.com/healthcare-it-solutions/picos/assets/132355030/7a9b5342-f2a4-478c-b04f-307666958718)